CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 
 ## Features
 
+- Renamed "electrode diffusivity" to "particle diffusivity" as a non-breaking change with a deprecation warning ([#3624](https://github.com/pybamm-team/PyBaMM/pull/3624))
```

Comment:
Can you move this to the unreleased section?

Reply:
Sure, thank you for reviewing.